language/testing-infra/e2e-tests/src/versioning.rs

```diff
@@ -41,7 +36,23 @@ impl VersionedTestEnv {
                     .payload(release_1_2_0_writeset())
                     .sign(),
             );
-            dr_sequence_number = 2;
+            dr_sequence_number += 1;
+        }
+
+        if version_number > 2 {
+            executor.execute_and_apply(
+                dr_account
+                    .transaction()
+                    .sequence_number(2)
+                    .payload(release_1_4_0_writeset())
```

Comment:
Not a blocker for this release, but do you think we should backfill the 1.3 writeset here as well?